rdt/errors.py

```diff
@@ -7,3 +7,7 @@ class NotFittedError(Exception):
 
 class Error(Exception):
     """Error to raise when ``HyperTransformer`` produces a controlled error message."""
+
+
+class InvalidSdtypeForTransformerError(Exception):
```

Comment:
I think we should change this error name to match the ones we created in the doc. You won't need to catch this error anymore in the integration tests if you do the other change